### DISCUSSION TITLE
It's possible to get SS without having to get every slider ends

### DISCUSSION THREAD

**Discussion by DarkbitNetwork** [ppy/osu#25452] [Q&A]
Due to the new slider mechanic for non-classic plays where dropping the slider end will not give combo or score (but will not break combo), it's now possible to get SS without actually having to get every slider end.

This is not a problem in stable because dropping slider ends will make the slider give 100s and also not a problem in 2023.1026.0 and earlier because dropping the slider end will reduce your accuracy VERY slightly in a way that it's unnoticeable for non-SS play(s) but impossible to get 100% accuracy if you drop any slider end.

Although unrelated to the topic being discussed, slider ends now count as slider ticks in the score card but dropping the slider end also reduces the **maximum** slider ticks? I'm not sure if I'll like these changes.

Not sure if this should be a discussion or an issue. But I decided to put it here because I don't think this is considered as a bug.

### Screenshots or videos
This is done on a beatmap where there's only 2 sliders (1st without slider tick, and 2nd with one), 
[Chipzel - Focus (Gamebits).osz.zip](https://github.com/ppy/osu/files/13353764/Chipzel.-.Focus.Gamebits.osz.zip) remove the .zip extension

**A play without any slider ends dropped**

![actual ss](https://github.com/ppy/osu/assets/43719333/43f953d6-e2bf-4476-abda-b8773aff6854)

**A play with every slider ends dropped**

![uhhh](https://github.com/ppy/osu/assets/43719333/20288068-0e97-49d9-9ef4-2b0fa2bcd9b1)

**Comment by bdach** [comment]
> but dropping the slider end also reduces the maximum slider ticks

that's a bug, see https://github.com/ppy/osu/issues/25464
